templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-3">
+              <h3 class="p-heading--5">Custom Internet of Things images for Intel</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Advanced optimizations and capability support are available for Intel network and edge platforms. Canonical offers support through custom IoT images for Ubuntu Desktop, Ubuntu Server and Ubuntu Core.
+              </p>
+              <div class="p-cta-block">
+                <a href="/download/iot/intel-iot">Download Ubuntu for Intel IoT platforms&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-3">
+              <h3 class="p-heading--5">Real-time Ubuntu for Intel SoCs</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Ensure that high-priority processes are executed first with Real-time Ubuntu. Canonical and Intel collaborate to deliver real-time for Ubuntu 22.04 on 12th Gen Intel® Core™ processors with TSN and TCC Support.
+
+              </p>
+              <div class="p-cta-block">
+                <a href="/real-time">Explore how real-time benefits IoT use cases&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-3">
+              <h3 class="p-heading--5">Hardware enablement (HWE) stacks</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                The HWE stack includes a rolling kernel for the most recent Ubuntu LTS release, so that users can take advantage of new hardware platform support of newer kernels. The final HWE kernel for an LTS release is the same as the next LTS release, and receives the LTS commitment.
+              </p>
+              <div class="p-cta-block">
+                <a href="/kernel/lifecycle">Learn more about the HWE lifecycle&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3 col-medium-3">
+              <h3 class="p-heading--5">Ubuntu Releases</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                The optimization and feature support for Intel platforms intercepts future Ubuntu releases, including images that receive up to 12 years of long term support with the Ubuntu Pro subscription.
+              </p>
+              <div class="p-cta-block">
+                <a href="/download">Download Ubuntu&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="u-fixed-width">
```

Comment:
This section should be separated from the previous one for usability purposes. On the copy docs this image is related to the next section, rather than the previous one. Might be confusing if other devs work on this in the future.

Reply:
@mattea-turic Could you help me out here, if this image is supposed to be a different section

Reply:
The image should be in the section "Benefit from support for the latest hardware", if that's what you're asking? Agreed with the PM on this – the copydoc needs to be updated